linode_api4/groups/object_storage.py

```diff
@@ -0,0 +1,649 @@
+        :type acl: str
+                   Enum: private,public-read,authenticated-read,public-read-write
+
+        :param cluster: The ID of the Object Storage Cluster where this bucket
+                        should be created.
+        :type cluster: str
+
+        :param cors_enabled: If true, the bucket will be created with CORS enabled for
+                             all origins. For more fine-grained controls of CORS, use
+                             the S3 API directly.
+        :type cors_enabled: bool
+
+        :param label: The name for this bucket. Must be unique in the cluster you are
+                      creating the bucket in, or an error will be returned. Labels will
+                      be reserved only for the cluster that active buckets are created
+                      and stored in. If you want to reserve this bucket’s label in
+                      another cluster, you must create a new bucket with the same label
+                      in the new cluster.
+        :type label: str
+
+        :returns: A Object Storage Buckets that created by user.
+        :rtype: ObjectStorageBucket
+        """
+        cluster_id = (
+            cluster.id if isinstance(cluster, ObjectStorageCluster) else cluster
+        )
+
+        params = {
+            "cluster": cluster_id,
+            "label": label,
+            "acl": acl,
+            "cors_enabled": cors_enabled,
+        }
+
+        result = self.client.post("/object-storage/buckets", data=params)
+
+        if not "label" in result or not "cluster" in result:
+            raise UnexpectedResponseError(
+                "Unexpected response when creating Object Storage Bucket!",
+                json=result,
+            )
+
+        return ObjectStorageBucket(
+            self.client, result["label"], result["cluster"], result
+        )
+
+    def object_acl_config(self, cluster_id, bucket, name=None):
+        """
+        View an Object’s configured Access Control List (ACL) in this Object Storage
+        bucket. ACLs define who can access your buckets and objects and specify the
+        level of access granted to those users.
+
+        This endpoint is available for convenience.
+        It is recommended that instead you use the more fully-featured S3 API directly.
+
+        API Documentation: https://www.linode.com/docs/api/object-storage/#object-storage-object-acl-config-view
+
+        :param cluster_id: The ID of the cluster this bucket exists in.
+        :type cluster_id: str
+
+        :param bucket: The bucket name.
+        :type bucket: str
+
+        :param name: The name of the object for which to retrieve its Access Control
+                     List (ACL). Use the Object Storage Bucket Contents List endpoint
+                     to access all object names in a bucket.
+        :type name: str
+
+        :returns: The Object's canned ACL and policy.
+        :rtype: dict { acl: str, acl_xml: str }
```

Comment:
Since returning object isn't a `dict` nor a subclass of `dict`, I would recommend this way to organize the doc for return value.
```suggestion
        :returns: The Object's canned ACL and policy. The returning MappedObject contains two string attributes: `acl` and `acl_xml`.
        :rtype: MappedObject
```